torch/linalg/__init__.py

```diff
@@ -1212,19 +1267,24 @@
 """)
 
 cond = _add_docstr(_linalg.linalg_cond, r"""
-linalg.cond(input, p=None, *, out=None) -> Tensor
+linalg.cond(A, p=None, *, out=None) -> Tensor
 
-Computes the condition number of a matrix with respect to a norm. Supports batched inputs.
+Computes the condition number of a matrix with respect to a norm.
 
-The condition number :math:`\kappa` of a matrix :math:`A` is defined as
+Letting :math:`\mathbb{K}` be :math:`\mathbb{R}` or :math:`\mathbb{C}`,
+The condition number :math:`\kappa` of a matrix
+:math:`A \in \mathbb{K}^{n \times n}` is defined as
 
 .. math::
 
     \kappa(A) = \|A\|_p\|A^{-1}\|_p
 
-The condition number of `A` measures the numerical stability of the linear system `AX = B`
+The condition number of :attr:`A` measures the numerical stability of the linear system `AX = B`
 in some specific matrix norm.
```

Comment:
I'm not in love with the language "in some specific matrix norm." Is numerical stability often "in a norm"?

Reply:
The numerical stability of a system is computed with respect to a norm.

I'm changing the wording to
> ... of the linear system `AX = B` with respect to a matrix norm.